ragaai_catalyst/__init__.py

```
@@ -9,6 +9,7 @@
 from .guardrails_manager import GuardrailsManager
 from .guard_executor import GuardExecutor
 from .tracers import Tracer, init_tracing, trace_agent, trace_llm, trace_tool, current_span, trace_custom
+from .redteaming import RedTeaming
```

Ruff (0.8.2)

Redefinition of unused `RedTeaming` from line 8

Remove definition: `RedTeaming`

(F811)